renderer/plugin.js

```diff
@@ -294,6 +299,18 @@ org.ekstep.questionsetRenderer = IteratorPlugin.extend({ // eslint-disable-line
       return this._renderedQuestions[renderIndex + 1];
     }
   },
+  updateMaxScore: function(shuffle, question){
+    // Update max-score of the question, when shuffle on
+    if(shuffle){
+      questionConfigData  = JSON.parse(question.config.__cdata);
+      questionConfigData.max_score = 1;
+      questionConfigData.metadata.max_score = 1;
```

Comment:
@shivashanmugam Do undefined check of metadata before assigning to max_score. This is failing for V1 questions.